app/models/miq_alert.rb

```diff
@@ -26,6 +26,7 @@ class MiqAlert < ApplicationRecord
     ExtManagementSystem
     MiqServer
     ContainerNode
+    ContainerProject
```

Comment:
You should add a miq_alert_statuses relation to container_project.rb
```
has_many :miq_alert_statuses, :as => :resource, :dependent => :destroy
```

Generally looks good, It would be helpful if you could separate the pieces related to the new event from the policy/alert bits.

Reply:
Added the relation but I didnt split the PR since this is a small feature that already has 3 PRs associated with it.
I think its a little too much noise.

Reply:
This change caused ui-classic travis failures, fixed in https://github.com/ManageIQ/manageiq-ui-classic/pull/3382